homeassistant/components/synology_dsm/strings.json

```diff
@@ -10,6 +10,9 @@
           "verify_ssl": "[%key:common::config_flow::data::verify_ssl%]",
           "username": "[%key:common::config_flow::data::username%]",
           "password": "[%key:common::config_flow::data::password%]"
+        },
+        "data_description": {
+          "host": "The hostname or IP address of your Synology nas."
```

Comment:
```suggestion
          "host": "The hostname or IP address of your Synology NAS."
```

Reply:
yeah you're right ... it is a shorthand, not a noun 👍